cmd/exo/cmd/privnet.go

```diff
@@ -34,3 +37,23 @@ func getNetworkByName(name string) (*egoscale.Network, error) {
 func init() {
 	RootCmd.AddCommand(privnetCmd)
 }
+
+// dhcpRange returns the string representation for a DHCP
+func dhcpRange(startIP, endIP, netmask net.IP) string {
+	if startIP != nil && endIP != nil && netmask != nil {
+		mask := net.IPMask(netmask.To4())
+		prefixSize, _ := mask.Size()
+		return fmt.Sprintf("%s-%s /%d", startIP.String(), endIP.String(), prefixSize)
+	}
+	return "n/a"
+}
+
+func netmaskString(ip net.IP, mask uint) net.IP {
```

Comment:
this is confusing: `func netmaskString -> net.IP`, return a string.

Reply:
Updated